src/gui/tray/UserModel.h

```diff
@@ -52,6 +52,7 @@ class User : public QObject
     void hasLocalFolderChanged();
     void serverHasTalkChanged();
     void avatarChanged();
+    void accountStateChanged(int state);
```

Comment:
Regarding my previous comment about the onRefreshCurrentUserGui that'd be likely by having UserModel reacting to signals in here that this would get implemented.